Resources/Private/Templates/Backend/Header/HeaderInfo.html

```
@@ -53,12 +53,14 @@
     <div class="content-planner-header__actions">
         <div class="content-planner-header__ce-hint">
             <f:for each="{contentElements}" as="contentElement">
+                <f:variable name="ceStatusTitle" value="{xcp:statusTitle(statusId: contentElement.tx_ximatypo3contentplanner_status)}"/>
+                <f:variable name="ceHintLabel"><f:translate key="header.editContentElement" extensionName="xima_typo3_content_planner" arguments="{0: contentElement.title}"/><f:if condition="{ceStatusTitle}"> (<f:translate key="header.status" extensionName="xima_typo3_content_planner"/>: {ceStatusTitle})</f:if></f:variable>
                 <be:link.editRecord
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)